xapian-bindings/configure.ac

```diff
@@ -287,11 +291,31 @@ case $with_python$with_python3$with_php7$with_ruby$with_tcl$with_csharp$with_jav
   test -z "$with_ruby" && with_ruby=no
   test -z "$with_perl" && with_perl=no
   test -z "$with_lua" && with_lua=no
+  test -z "$with_go" && with_go=no
   ;;
 esac
 
 BINDINGS=
-
+if test no != "$with_go" ; then
+  AC_PATH_PROG([GO],["go"],[])
+  AC_MSG_CHECKING(go)
+  if test -n "$GO" ; then
+    AC_MSG_RESULT(yes)
+    AC_MSG_CHECKING(goroot)
+    GOROOT=`go env GOROOT`
+    AC_SUBST(GOROOT)
+    if test -n "${GOROOT}" ; then
+      AC_MSG_RESULT([${GOROOT} yes])
+    fi
+    if test -z "${GOROOT}" ; then
+      AC_MSG_ERROR([goroot not set correctly])
+    fi
+  fi
+  BINDINGS="$BINDINGS go"
+  if test -z "$GO" ; then
+    AC_MSG_ERROR("go not installed or go path has not been set")
+  fi
+fi
```

Comment:
You've added entries for Go last above and below (which seems reasonable to me as that's how previous new languages have been added) but here you've added it as the **first** language probed for.  I think a consistent order (i.e. after Lua) would make this easier to maintain.

Reply:
yea sure ill replace it